src/middlewares/user.verifyEmail.middleware.js

```diff
@@ -0,0 +1,20 @@
+import { User } from '../database/models/index';
+import { JwtUtility } from '../utils/jwt.util.js';
+
+export const verifyUserEmail = async(req, res, next) => {
+  const { token } = req.params
+  const decoded = JwtUtility.verifyToken(token);
+  if(decoded.hasOwnProperty('email')){
+    const getUser = await User.findOne({
+      where: { email: decoded.email },
+    });
+    if(getUser){
+        getUser.verified = true;
+        getUser.save();
+        req.user = decoded;
+        next();
+    }
+  } else {
+    res.status(403).json({message: 'Failed to verify your email'});
+  }
+}
```

Comment:
- This middleware doesn't make sense
- Verification is a user-related task. you don't need to create new file